pom.xml

```diff
@@ -160,6 +160,11 @@
             <groupId>org.springframework.cloud</groupId>
             <artifactId>spring-cloud-stream-binder-rabbit</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.awspring.cloud</groupId>
+            <artifactId>spring-cloud-aws-starter-s3</artifactId>
+            <version>3.0.1</version>
```

Comment:
3.1 (dec 2023) and 3.2 (sept 2024) have been released in the meantime

Reply:
https://github.com/awspring/spring-cloud-aws

Reply:
Go for 3.2 or it's a bit early ?